src/main/java/org/jnode/fs/xfs/attribute/XfsAttribute.java

```diff
@@ -0,0 +1,133 @@
+package org.jnode.fs.xfs.attribute;
+
+import org.jnode.fs.xfs.XfsObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * A XFS SELinux Attribute.
```

Comment:
This isn't just for the SELinux attribute is it?

Reply:
Hi Erwin, no,  seems that this class should be used to work with the extent attribute. So I updated the title to extended attribute.